osc_sdk/sdk.py

```diff
@@ -134,9 +135,10 @@ def method(self):
 
     @method.setter
     def method(self, method):
-        if method not in {'GET', 'POST'}:
+        if method not in METHODS_SUPPORTED:
```

Comment:
Alors c'est tatillons, mais est-ce que SUPPORTED_METHODS ne serait pas plus lisible ?
`if method not in SUPPORTED_METHODS:` ??

Reply:
Si tu veux chercher dans les constants, tu va chercher par l'objet. c'est la raison que j'ai nommé comme ça

Reply:
Okay